### PR TITLE
(docs): add new changes to how to mint

### PR DESCRIPTION
**Pull request by hanakoori01**
### ¿Qué hace este PR?

- Finalizar ¿Cómo mintear NFTs?

### Pasos 
1. Correr `yarn build`
2. Correr `yarn serve`
3. Ir a `localhost:3000`
4. Ir a Pautas para Desarrolladores > Tutoriales > ¿Cómo acuñar NFTs?

### CODE REVIEW

File: docs/tutoriales/como-mintear.mdx [hunk 1]
```diff
@@ -1,80 +1,121 @@
 ---
 id: como-mintear
-title: ¿Cómo mintear NFTs?
-sidebar_label: ¿Cómo mintear NFTs?
+title: ¿Cómo acuñar NFTs?
+sidebar_label: ¿Cómo acuñar NFTs?
```
Comment (leisterfrancisco) [inline docs/tutoriales/como-mintear.mdx:4]: I don't think `acuñar` is the best translation

File: docs/tutoriales/como-mintear.mdx [hunk 2]
```diff
@@ -1,80 +1,121 @@
...
-]
+  {
+    instruction:
+      "Ir a Areaxnft.com para poder conectar nuestra billetera, en este caso hemos seleccionado Anchor Wallet.",
+    imageSrc: "img/como-mintear/telos_1.png",
+    href: "#areax-marketplace",
+  },
+  {
+    instruction:
+      "Una vez que haya conectado su billetera, hará clic en Crear NFT. Seleccione el tipo de conjunto que necesita. La cantidad de $ TLOS que necesitará para acuñar dependerá de la cantidad de NFT que vaya a subir.",
```
Comment (leisterfrancisco) [inline docs/tutoriales/como-mintear.mdx:58]: take a look at `hará`

File: docs/tutoriales/como-mintear.mdx [hunk 3]
```diff
@@ -1,80 +1,121 @@
...
-    }
-]
+  {
+    instruction:
+      "Ir a wax.atomichub.io para poder conectar nuestra billetera, en este caso hemos seleccionado Anchor Wallet.",
+    imageSrc: "img/como-mintear/wax_1.png",
+    href: "#atomichub",
+  },
+  {
+    instruction: "Click en crear nueva colección",
```
Comment (leisterfrancisco) [inline docs/tutoriales/como-mintear.mdx:19]: Change `Click` to `Clic`

More on [RAE Clic](https://www.rae.es/dpd/clic)

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/tutoriales/como-mintear.mdx (hunk 1 above)
Comment (hanakoori01) [reply docs/tutoriales/como-mintear.mdx:4]: Leo told me to change `mintear` for `acuñar` so i don't really know what word fits best

Comment (hanakoori01) [reply docs/tutoriales/como-mintear.mdx:4]: image

Comment (hanakoori01) [reply docs/tutoriales/como-mintear.mdx:4]: Perfect

File: docs/tutoriales/como-mintear.mdx [hunk 4]
```diff
@@ -55,19 +55,19 @@ export const areaxStepper = [
   },
   {
     instruction:
-      "Una vez que haya conectado su billetera, hará clic en Crear NFT. Seleccione el tipo de conjunto que necesita. La cantidad de $ TLOS que necesitará para acuñar dependerá de la cantidad de NFT que vaya a subir.",
+      "Una vez que haya conectado su billetera, haga clic en «Create NFT». Seleccione el tipo de conjunto que necesita. La cantidad de $ TLOS que necesitará para mintear dependerá de la cantidad de NFT que desee subir.",
```
Comment (leisterfrancisco) [inline docs/tutoriales/como-mintear.mdx:58]: What if we stick to Markdown syntax to highlight wathever section we want to the reader?
For example:

`Create NFT` or **Create NFT**

The same thing applies to the rest of the content.

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**